test/unit/models/test/TestPreviewRequestTest.php

```
@@ -42,5 +42,12 @@ public function testConstruct(): void
     {
         $this->assertInstanceOf(TestPreviewConfig::class, $this->subject->getConfig());
         $this->assertSame(self::TEST_URI, $this->subject->getTestUri());
+        $this->assertFalse($this->subject->isTimeConstraintRequired());
+    }
+
+    public function testIsTimeConstraintRequired(): void
+    {
+        $subject = new TestPreviewRequest(self::TEST_URI, isTimeConstraintRequired: true);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$subject'. (undefined)

(UnusedLocalVariable)